custom pages example and an email adapter is added (#5838)
parse-community/parse-server: README.md

```

You can also use other email adapters contributed by the community such as:

> - [parse-smtp-template (Multi Languaje and Multi Template)](https://www.npmjs.com/package/parse-smtp-template)

Reviewer: ```suggestion
- [parse-smtp-template (Multi Language and Multi Template)](https://www.npmjs.com/package/parse-smtp-template)
```

Author: Thanks :)

> - [parse-smtp-template (Multi Languaje and Multi Template)](https://github.com/macarthuror/parse-smtp-template)

Reviewer 2: Can you add the npm link instead?

Author: Sure 👌

- [simple-parse-smtp-adapter](https://www.npmjs.com/package/simple-parse-smtp-adapter)
- [parse-server-generic-email-adapter](https://www.npmjs.com/package/parse-server-generic-email-adapter)

### Custom Pages

> Parse server can let you  change the default pages of the app and redirect the user to other path or domain.

Reviewer: ```suggestion
It’s possible to change the default pages of the app and redirect the user to another path or domain.
```
